src/CwpSearchResult.php

```diff
@@ -200,7 +200,7 @@ protected function getLink($terms, $format = null)
         if (!$terms) {
             return null;
         }
-        $link = 'search/SearchForm?Search='.rawurlencode($terms ?? '');
+        $link = '/search/SearchForm?Search='.rawurlencode($terms ?? '');
```

Comment:
This would break on sites where the homepage is not the root of the domain e.g.

`www.mysite.com/my-silverstripe-site/my-page`

I think that prefixing this with `Director::absoluteBaseURL()` instead of just `'/'` is correct?